From: Colin Ian King <colin.king@canonical.com>
Date: Tue, 4 Oct 2016 12:20:16 +0100
Subject: i40e: Add missing \n to end of dev_err message
Patch-mainline: v4.10-rc1
Git-commit: ff00f3a9670f97203b33ac00eeca81d75768919e
References: bsc#1024346 FATE#321239 bsc#1024373 FATE#321247

Trival fix, dev_err message is missing a \n, so add it.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Tested-by: Andrew Bowers <andrewx.bowers@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/ethernet/intel/i40e/i40e_virtchnl_pf.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/intel/i40e/i40e_virtchnl_pf.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_virtchnl_pf.c
@@ -2933,7 +2933,7 @@ int i40e_ndo_set_vf_bw(struct net_device
 	}
 
 	if (max_tx_rate > speed) {
-		dev_err(&pf->pdev->dev, "Invalid max tx rate %d specified for VF %d.",
+		dev_err(&pf->pdev->dev, "Invalid max tx rate %d specified for VF %d.\n",
 			max_tx_rate, vf->vf_id);
 		ret = -EINVAL;
 		goto error;
